services/data-info/src/data_info/routes/schemas/data.clj

```diff
@@ -9,6 +9,13 @@
         [heuristomancer.core :as info])
   (:require [schema.core :as s]))
 
+(s/defschema PathToUUIDParams
+  (assoc StandardUserQueryParams
+         :path (describe NonBlankString "A path to translate to a UUID")))
+
+(s/defschema PathToUUIDReturn
+  {:id (describe NonBlankString "The UUID for the path queried.")})
```

Comment:
This could be a `UUID` instead of `NonBlankString`, then the service function wouldn't have to use `str` on the return value first?